cstore_writer.c

```diff
@@ -924,71 +1047,99 @@ AppendStripeMetadata(TableFooter *tableFooter, StripeMetadata stripeMetadata)
 }
 
 
-/* Writes the given data to the given file pointer and checks for errors. */
+/*
+ * WriteToFile appends provided data to the active page. If data does
+ * not fit into remaining space in page, a new page is allocated until whole
+ * data is written.
+ */
 static void
-WriteToFile(FILE *file, void *data, uint32 dataLength)
+WriteToFile(TableWriteState *writeState, void *data, uint32 dataLength)
```

Comment:
needs refactoring. I would like to make a function something like WriteToFork and call this function to write both data and footer